src/main.py

```diff
@@ -196,7 +215,7 @@ def _update_opt_in_status(tid: str, new_opt_in_status: bool):
         app_name = 'opt_in_out'
         app_version = Telemetry.get_version()
         opt_in_checker = OptInChecker()
-        opt_in_check = opt_in_checker.check()
+        opt_in_check = opt_in_checker.check(enable_opt_in_dialog=False)
```

Comment:
can you add a comment why it is switched off here?

Reply:
Added comment.